test/functional/README.md

```diff
@@ -51,10 +51,13 @@ don't have test cases for.
 
 #### General test-writing advice
 
+- Instead of inline comments or no test documentation at all, log the comments to the test log, e.g.
+  `self.log.info('Create enough transactions to fill a block'`. Logs make the test code easier to read and the test
```

Comment:
nit: does this need a closing `)`

Reply:
good eye

Reply:
Added `)`, and changed commit hash from eeee... to ffff...